src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepository.kt

```diff
@@ -1,12 +1,49 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProvider
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProviderSentReferralSummary
 import java.util.UUID
 
 interface ReferralRepository : JpaRepository<Referral, UUID> {
+  @Query(
+    value = """select sentAt,
+			referenceNumber,
+			assignedToUserName,
+			interventionTitle,
+			serviceUserFirstName,
+			serviceUserLastName from (	
+	select
+			cast(r.sent_at as TIMESTAMP WITH TIME ZONE) as sentAt,
+			r.reference_number as referenceNumber,
+      dfc.id as dynamicFrameworkContractId,
+			au.user_name as assignedToUserName,
+			i.title as interventionTitle,
+			rsud.first_name as serviceUserFirstName,
+			rsud.last_name as serviceUserLastName,
+			row_number() over(partition by r.id order by ra.assigned_at desc) as assigned_at_desc_seq		
```

Comment:
Do we have a business rule to show assigned cases first?

If yes, could we use a simpler `ORDER BY`, please? `ORDER BY COALESCE(ra.assigned_at, r.sent_at) DESC`